### PR TITLE
Doc/2024/march/meetup

### PR DESCRIPTION
**Pull request by CandicePersil**
Django meetup of March 2024:

* Addition of the link to the official YouTube channel
* Creation of a new folder for March meetup
* Addition of the Django channel slides in the new folder

### CODE REVIEW

File: slides/2024/march/django-channels/index.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
+<!doctype html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<head>
```
Comment (sabderemane) [inline slides/2024/march/django-channels/index.html]: Oh it's with reveal.js, Paul didn't manage to export it in PDF or images?

File: slides/2024/march/django-channels/.DS_Store
Comment (sabderemane) [inline slides/2024/march/django-channels/.DS_Store:1]: This file shouldn't be here 👀

**Review by sabderemane — COMMENTED** [review]
Nice the addition of the youtube channel ⭐  I have some concern on the slides

File: slides/2024/march/django-channels/index.html (hunk 1 above)
Comment (CandicePersil) [reply slides/2024/march/django-channels/index.html]: You prefer it to be in PDF format ? I can try to export it.

File: slides/2024/march/django-channels/.DS_Store
Comment (pfouque) [reply slides/2024/march/django-channels/.DS_Store:1]: `https://github.com/github/gitignore/blob/main/Global/macOS.gitignore` =>  `~/.config/git/config`

File: slides/2024/march/django-channels/index.html (hunk 1 above)
Comment (sabderemane) [reply slides/2024/march/django-channels/index.html]: I think it's will be more easy if possible for people who don't want to run it in order to see the slides

**Review by sabderemane — APPROVED** [review]
LGTM  🚀

**Pull request merged**